tests/test_util.py

```diff
@@ -13,6 +13,7 @@
 import gcs_utils
 import resources
 
+TEST_RESOURCES_BUCKET = 'aou-res-curation-test-resources'
```

Comment:
@mark-velez can we store the bucket name in the environment variables? or is it okay to be here?

Reply:
Yes, I discussed this with Lauren- this is not great, but it is no worse than the current approach of hardcoding a file_id. I'm reluctant to add another environment variable. Let me know if you can think of any easy alternatives.

Reply:
@mark-velez  we have project_id already in envireonment_variables, we can leverage it '{project_id}-resources` . let me know your thoughts.